validator/slashing-protection/slasher_client.go

```diff
@@ -142,5 +144,8 @@ func (s *Service) Status() error {
 	if s.conn == nil {
```

Comment:
Comment right above this says WIP, can you update that please?

Reply:
sure